docker/build_scripts_pypy/prefetch_pypy.sh

```diff
@@ -17,7 +17,6 @@ cd "$SOURCES"
 fetch_source pypy-7.2.0-linux_x86_64-portable.tar.bz2 "$SQUEAKY_GITHUB_URL/pypy-7.2.0"
 fetch_source pypy3.6-7.2.0-linux_x86_64-portable.tar.bz2 "$SQUEAKY_GITHUB_URL/pypy3.6-7.2.0"
 
-# pypy 7.3.3
-fetch_source pypy2.7-v7.3.3-linux64.tar.bz2 "$URL"
-fetch_source pypy3.6-v7.3.3-linux64.tar.bz2 "$URL"
```

Comment:
should we keep the last release of pypy3.6 in the image for now as well?

Reply:
We could, yes. Currently it's also removed from joerick/cibuildwheel#629, and we didn't get any complaints from @mattip. If we keep it here, `cibuildwheel` should probably also keep it, right?

Up to the PyPy team to decide the policy on that, I would say. But it's just a patch version increase and still the same ABI tag, so it could make sense to me.

Reply:
This docker image is more general than cibuildwheel (which is usually only used to build wheels) so maybe it makes sense to be more cautious here and leave 3.6 in the image.

Reply:
Also fine for me. It still feels weird to have this mismatch (and increase the docker image's size), as manylinux is also mainly aimed at building wheels, I thought? But done, anyway!

Reply:
> increase the docker image's size

Actually, this only added around 30 MB to the docker image. No reason to worry about it, then. Apologies!